src/App.tsx

```diff
@@ -1,43 +1,294 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useEffect, useRef, useState,
+} from 'react';
+import classNames from 'classnames';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Todo } from './types/Todo';
+import { client } from './utils/fetchClient';
+import { TodoList } from './components/TodoList';
+import {
+  Error, ErrorTimeout, ErrorButtonHandler, ErrorCloser,
+} from './types/Error';
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>([]);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [needUpdate, setNeedUpdate] = useState(false);
+
```

Comment:
there is no need to add empty lines after each constant. Additional lines are required between different logical blocks. For example, declaration of constants + empty line + function. fix it all over the file